### ISSUE TITLE
chunks rate limit

### ISSUE THREAD

**Issue by shukryshuk** [mrrfv/cloudflare-gateway-pihole-scripts#131]
hello , can u add this to script to include an 8 second sleep after every 10 chunks. So, after each batch of 10 chunks, the script should pause for 8 seconds before continuing with the next batch. or maybe 10 seconds ??

**Comment by n0w33d** [comment]
Yeah I am also getting a bunch of HTTP 400 errors because Cloudflare is rate-limiting the uploads when ran locally.

Please help!
